.gitattributes

```diff
@@ -1,4 +1,6 @@
-*.pbxproj -diff linguist-generated
+*.pbxproj binary linguist-generated
+ios/Exponent.xcodeproj/project.pbxproj merge=mergepbx -text diff linguist-generated
```

Comment:
Thanks for applying my suggestion! I've inspected the `mergepbx` library and started to wonder if… this will work automatically? The setup on https://github.com/simonwagner/mergepbx#using-mergepbx-as-a-merge-driver suggests to also edit `~/.gitconfig` — I don't suppose we'd want to make all the contributors have to modify it manually? Or is it somehow automatically taken care of?

Reply:
Yeah, after a second thought I think I'm going to remove it.
Why?
1) You have to have it installed to be available
2) Even though it probably might come in handy (I believe every `pod install` modifies whole `.pbxproj` file and especially UUIDs are generated from scratch) and `union` merge would possibly fail miserably in almost every case, I don't think we need sth like this and I haven't tested it so ¯\_(ツ)_/¯